backend/inspirehep/migrator/tasks.py

```diff
@@ -190,16 +223,28 @@ def migrate_recids_from_mirror(
     logger.info(f"Processing {all_recs} records in this migration step")
     step = migration_steps[step_no]
     header = (step.s(r) for r in recids_chunks)
-    chord_task = chord(header)(
-        migrate_recids_from_mirror.s(
+    if one_step:
+        callback = stop_after_one_step.s()
```

Comment:
shouldn't this be 

```python
 callback = migrate_recids_from_mirror.s(
            step_no=step_no ,
            disable_orcid_push=disable_orcid_push,
            disable_orcid_push=disable_orcid_push,
            disable_references_processing=disable_references_processing,
            disable_references_processing=disable_references_processing,
        )
```

Reply:
No, why?

Reply:
my bad sorry I didn't follow the code correclty